test/test_ops.py

```diff
@@ -829,6 +837,22 @@ def test_batched_nms_implementations(self, seed):
         empty = torch.empty((0,), dtype=torch.int64)
         torch.testing.assert_close(empty, ops.batched_nms(empty, None, None, None))
 
+data_dependent_torchvision_test_checks = [
+    "test_schema",
+    "test_autograd_registration",
+    "test_faketensor",
+    "test_aot_dispatch_dynamic",
+]
+
+optests.generate_opcheck_tests(
+    TestNMS,
+    ["torchvision"],
+    {},
+    "test/test_ops.py",
+    [],
+    data_dependent_torchvision_test_checks,
+)
```

Comment:
Hmm, after https://github.com/pytorch/pytorch/pull/109110 the failures dict is no longer a python dict, generate_opcheck_tests now assumes the existence of a json file.

The NMS tests work on everything, so there are no expected failures. It seems unfortunate that we would need to have a .json file in the repo to use generate_opcheck_tests. But I don't really have a better idea right now, the reason why we require a json file is so that we can automatically update it by writing to it; a string is a bit more difficult because we haven't hooked into the expecttest mechanism.

Reply:
I don't mind having to have an empty JSON, but I am going to have to do this for each test class (of which there are a lot)